z.lua

```diff
@@ -1866,6 +1866,30 @@ if [ -n "$BASH_VERSION" ]; then
 fi
 ]]
 
+local script_fzf_complete_bash = [[
+if command -v fzf >/dev/null 2>&1; then
+	# To redraw line after fzf closes (printf '\e[5n')
+	bind '"\e[0n": redraw-current-line'
+
+	_zlua_fzf_complete() {
+		local query="${COMP_WORDS[COMP_CWORD]}"
+
+		fzf="fzf --reverse --inline-info +s --height 35%"
```

Comment:
`--height` is unavailable on cygwin and msys.

So I introduce an `$_ZL_FZF_FULLSCR` for this,

https://github.com/skywind3000/z.lua/blob/69ad680b689ac59331245819c631510d8f559bc9/z.lua#L1408-L1410

Could you handle it in `_zlua_fzf_complete` ??

Reply:
check 5cd9136, not sure if this is the right way to do it